### ISSUE TITLE
Sensible Golf (1994)

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoWin3x#47]
The mouse cursor is invisible. It you alt tab and then click back into the dosbox window you get the cursor back but as soon as you click on the game window inside dosbox it disappears again. This is the same in both daum and X.

krc found out a bit more info: Its to do with the cycles. Set them to 7000 and you will see the cursor flashing instead of invisible. The lower the cycles the less its invisible, however even 7000 cycles is bordering to slow to actually play.

**Comment by exoscoriae** [comment]
this game is picky in general. It only runs with machine set to pentium_slow or 486_slow and core set to auto.

I attempted it with different video drivers (et4000) with no luck.

I was also unable to even get the game to start in a vhd boot.

**Comment by Python-Exoproject** [comment]
Did you set the cycles to 7000 though so that its at least semi-playable?

**Comment by exoscoriae** [comment]
No. Even at 7,000 cycles I could only see the mouse when I first moved it. It wasn't any more useful than not being able to see it at all. Meanwhile, the game ran incredibly slow.
